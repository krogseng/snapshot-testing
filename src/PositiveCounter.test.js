import React from 'react';
import ReactDom from 'react-dom';
import PositiveCounter from './PositiveCounter';
import renderer from 'react-test-renderer';

it('should render 0', () => {
    const tree = renderer.create(<PositiveCounter />).toJSON();
    expect(tree).toMatchSnapshot();
});