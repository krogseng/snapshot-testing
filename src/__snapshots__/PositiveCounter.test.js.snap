exports[`test should render 0 1`] = `
<span>
  Value: 
  0
  <button
    className="decrement"
    onClick={[Function]}>
    âˆ’
  </button>
  <button
    className="increment"
    onClick={[Function]}>
    +
  </button>
</span>
`;
